src/pages/project/project_wizard/ProjectDetailsForm.tsx

```diff
@@ -1,10 +1,95 @@
-import { Box } from '@mui/material';
+import { Autocomplete, Box, TextField, Typography } from '@mui/material';
+import { Field, FieldProps, useFormikContext } from 'formik';
+import { Trans, useTranslation } from 'react-i18next';
 import { ErrorBoundary } from '../../../components/ErrorBoundary';
+import { cristinCategories } from '../../../resources/cristinCategories';
+import {
+  CristinProject,
+  ProjectFieldName,
+  ProjectOrganization,
+  SaveCristinProject,
+  TypedLabel,
+} from '../../../types/project.types';
+import { dataTestId } from '../../../utils/dataTestIds';
+import { getLanguageString } from '../../../utils/translation-helpers';
+import { OrganizationSearchField } from '../../basic_data/app_admin/OrganizationSearchField';
+import { isRekProject } from '../../registration/description_tab/projects_field/projectHelpers';
+import { ProjectFundingsField } from './ProjectFundingsField';
+import { FormBox } from './styles';
+
+interface ProjectDescriptionFormProps {
+  project: CristinProject;
+}
+
+export const ProjectDetailsForm = ({ project }: ProjectDescriptionFormProps) => {
+  const { t } = useTranslation();
+  const { values, setFieldValue, setFieldTouched, touched, errors } = useFormikContext<SaveCristinProject>();
+  const thisIsRekProject = isRekProject(project);
 
-export const ProjectDetailsForm = () => {
   return (
     <ErrorBoundary>
-      <Box sx={{ bgcolor: 'secondary.light', padding: '1.75rem 1.25rem' }}>test 2</Box>
+      <Box sx={{ display: 'flex', flexDirection: 'column', gap: '1rem' }}>
+        <FormBox>
+          <Typography variant="h2">{t('project.coordinating_institution')}</Typography>
+          <Field name={ProjectFieldName.CoordinatingInstitution}>
```

Comment:
Liten heads-up til fremtiden: Dette feltet skal også ha støtte for enhet, og ikke bare toppnivå-institusjon. Men bare å gjenbruke dette fra før enn så lenge!